.github/CODEOWNERS

```
@@ -490,6 +490,7 @@ Makefile                                              @thiagokokada
 /modules/services/xembed-sni-proxy.nix                @rycee
 
 /modules/services/xidlehook.nix                       @dschrempf
+/modules/services/xidlehook.nix                       @bertof
```

Can you add your name besides `@dschrempf`?
```suggestion
/modules/services/xidlehook.nix                       @dschrempf @bertof
```